conf/securesocial.conf

```diff
@@ -161,6 +161,15 @@ securesocial {
         #groups=["cn=org_isda,ou=Groups,dc=ncsa,dc=illinois,dc=edu"]
     }
 
+    keycloak {
+        authorizationUrl="http://localhost:8080/keycloak/realms/clowder1/protocol/openid-connect/auth"
```

Comment:
can we just make this `clowder` instead of clowder1

Reply:
fixed